src/frontend/src/lib/components/dapps/DappsExplorerSignedIn.svelte

```diff
@@ -74,3 +74,11 @@
 </ul>
 
 <SubmitDappButton />
+
+<style lang="postcss">
+	@media (max-width: 360px) {
+		ul {
+			@apply grid-cols-1;
+		}
+	}
+</style>
```

Comment:
isn't the minimum size `375px` (for example iPhone SE)? what is the use case for `360px` ? if there is, can we move it to tailwind? maybe defining a new screen

```
screens: {
  xs: '360px',
  ...
 }
```

Reply:
As far as I know, for mobile designs it would be best practice to use 320px as smallest possible size where your designs should make sense. Just to give one reference: https://www.browserstack.com/guide/responsive-design-breakpoints 
![image](https://github.com/user-attachments/assets/0ee087b0-c7e5-4826-9261-a751952d5404)

I added it for this special case because of a request from the design team:
![image](https://github.com/user-attachments/assets/eb89ff23-d81c-4e6c-8238-60f207b56e96)

I'm unsure if I should add it to the tw definitions because the rest of the application is fine with the current breakpoints. Seems like an exception to me for now.

Reply:
wow 320px! very small! thank you for the explanation!

I am just concerned, because the rest of the layout/interface doesn't respect this size, so I am not sure, if we should or not. Meaning, it seems strange to have this kind of check only in a single place in the entire OISY, you know what I mean?

Reply:
Yes, yes, agreed! But that is exactly my point: if the rest is not focused on 320px-screens, why should this one be? If there is a reason for this one to be treated for smaller screens, for sure there is the same reason for all the other screens/layouts, no? Am I missing something?

Reply:
Hm I try to outline the problem a bit. We normally style the application the same for 0-640px (sm breakpoint from tailwind). In this particular case, the requirement is to have a more fine-grained control over the grid. While two columns are fine for small devices > 360px, even smaller screens will break the cards content. Problem: 
<img width="581" alt="image" src="https://github.com/user-attachments/assets/bde9df64-6644-42e6-adba-73509920480b">

As this is a very specific case, I do not want to add this to a general configuration.